streeteasy-scraper.md

```diff
@@ -0,0 +1,15 @@
+This is a webscraper idea for the website steeteasy.com, which is a real estate platform similar to zillow.com but specific to new york city.
```

Comment:
move to `projects/javascript/streeteasy-scraper.md` (or whatever language you're using)

Reply:
Done.